bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,37 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        output_format='regex',
+        output_regex=r'(?P<message>\S+.*)')
+class LicenseCheckBear:
+    """
+    Attempts to determine the license that applies to each file passed to
+    it, by searching the start of the file for text belonging to various
+    licenses.
+    """
+    LANGUAGES = {'ALL'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='devscripts')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'License', 'Copyright Notice'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         licensecheck_lines: int=60,
```

Comment:
Just curious, why the default value for `licensecheck_lines` is 60? :sweat_smile: